spec/index.html

```diff
@@ -80,6 +80,21 @@ <h1>Map.prototype.filter( _callbackfn_, [ _thisArg_ ] )</h1>
     <emu-import href="./map-prototype-filter.html"></emu-import>
 </emu-clause>
 
+<emu-clause id="Map.prototype.find">
+    <h1>Map.prototype.find( _callbackfn_ [ , _thisArg_ ] )</h1>
+    <emu-import href="./map-prototype-find.html"></emu-import>
+</emu-clause>
+
+<emu-clause id="Map.prototype.findKey">
+    <h1>Map.prototype.findKey( _callbackfn_ [ , _thisArg_ ] )</h1>
+    <emu-import href="./map-prototype-find-key.html"></emu-import>
+</emu-clause>
+
+<emu-clause id="Map.prototype.keyOf">
+    <h1>Map.prototype.keyOf( _searchElement_ )</h1>
```

Comment:
bikeshed: i'd expect this to be called `keyFor`, both for symmetry with `Symbol`, and also it sounds more englishy to me (although i do see the symmetry with `indexOf` as well)